sklearn/utils/testing.py

```diff
@@ -697,5 +705,27 @@ def check_skip_travis():
     if os.environ.get('TRAVIS') == "true":
         raise SkipTest("This test needs to be skipped on Travis")
 
+class TempMemMap(object):
```

Comment:
As the name of the memory map type is `numpy.memmap` I would name this `TempMemmap`.